openpype/lib/path_tools.py

```diff
@@ -2,8 +2,12 @@
 import re
 import logging
 
+from openpype.api import Anatomy
+
 log = logging.getLogger(__name__)
 
+pattern_array = re.compile(r"\[.*\]")
```

Comment:
I would move this to first line of `compute_paths`.

Reply:
done